plugins/main/windows/win-bridge/win-bridge_windows.go

```diff
@@ -215,5 +215,5 @@ func cmdCheck(_ *skel.CmdArgs) error {
 }
 
 func main() {
-	skel.PluginMain(cmdAdd, cmdCheck, cmdDel, version.PluginSupports("0.1.0", "0.2.0", "0.3.0"), bv.BuildString("win-bridge"))
+	skel.PluginMain(cmdAdd, cmdCheck, cmdDel, version.PluginSupports("0.1.0", "0.2.0", "0.3.0", version.Current()), bv.BuildString("win-bridge"))
```

Comment:
this should be `version.All` rather than a list.

Reply:
fixed